core/domain/user_services_test.py

```diff
@@ -335,7 +409,51 @@ def test_get_user_settings_by_auth_id_strict_returns_user_settings(self):
     def test_get_user_settings_by_auth_id_strict_for_missing_auth_id_is_none(
             self):
         with self.assertRaisesRegex(Exception, 'User not found.'):
-            user_services.get_user_settings_by_auth_id('auth_id_x', strict=True)
+            user_services.get_user_settings_by_auth_id(
+                'auth_id_x',
+                strict=True
+        )
```

Comment:
Indent by 4 to match opening paren. (Or better still, keep this whole thing as it used to be.)